src/index.js

```diff
@@ -1,9 +1,20 @@
 import { Selector } from 'testcafe';
 
-export default Selector(complexSelector => {
-    function validateSelector (selector) {
+export default Selector((vueSelector, vueReference, rootElementReference) => {
+
+    function validateParameters (selector, reference, rootReference) {
         if (selector !== void 0 && typeof selector !== 'string')
             throw new Error('If the selector parameter is passed it should be a string, but it was ' + eval('typeof selector')); // eslint-disable-line no-eval
+        
+        if (reference) {
```

Comment:
this is intentional.
reference is optional and it can take false/null in case we want to ignore reference and provide only root reference.

it needs to string only if ref value is provided

i am checking whether ref value that is passed is string or not.